chainntnfs/txnotifier.go

```diff
@@ -633,7 +633,34 @@ func (n *TxNotifier) RegisterConf(txid *chainhash.Hash, pkScript []byte,
 		confSet = newConfNtfnSet()
 		n.confNotifications[ntfn.ConfRequest] = confSet
 	}
-	confSet.ntfns[ntfn.ConfID] = ntfn
+
+	// In case of an address reuse, we would get multiple confirmation
```

Comment:
Yeah that's what I also had in mind, adding the small check in `handleConfDetailsAtTip`. I don't think any changes need to be done for registrations, the assumption there is that it'll dispatch on the first transaction that satisfies it starting from the initial height hint.

Reply:
Ok, it would have been very simple if I wasn't looking in the wrong place the whole time... Sorry about that.
This indeed doesn't make it necessary to change anything with the registrations.